client/my-sites/checklist/wpcom-checklist/index.jsx

```diff
@@ -111,9 +111,18 @@ export default connect( ( state, ownProps ) => {
 	const designType = getSiteOption( state, siteId, 'design_type' );
 	const isEligibleForChecklist = isEligibleForDotcomChecklist( state, siteId );
 	const isSection = isSiteSection( state );
-	const taskStatuses = get( getSiteChecklist( state, siteId ), [ 'tasks' ] );
+	const siteChecklist = getSiteChecklist( state, siteId );
+	const siteSegment = get( siteChecklist, 'segment' );
```

Comment:
Site segment is no longer stored as a string representative of the site type, but as an id. You could do something like this:

`const siteSegment = getSiteTypePropertyValue( 'id', parseInt( get( siteChecklist, 'segment' ) ), 'slug' );`

or change the condition in [wpcom-task-list.js](https://github.com/Automattic/wp-calypso/pull/30213/files#diff-104d8425fbdae54125db07d46cb7168bR31) to something like:

`if ( 1 == siteSegment ) {`

The return type is a string however (hence the weak check and `parseInt` above), so that might changed in the backend (to an int)

Reply:
Thanks. I've inserted a line of code to get the segment slug from the segment ID the backend returns:
https://github.com/Automattic/wp-calypso/pull/30213/files#diff-104d8425fbdae54125db07d46cb7168bR19